bolt_unix.go

```diff
@@ -78,9 +78,29 @@ func munmap(db *DB) error {
 	}
 
 	// Unmap using the original byte slice.
-	err := syscall.Munmap(db.dataref)
+	err := unix.Munmap(db.dataref)
 	db.dataref = nil
 	db.data = nil
 	db.datasz = 0
 	return err
 }
+
+// mlock locks memory of db file
+func mlock(db *DB, fileSize int) error {
```

Comment:
I assume that the code currently would not compile for windows & other osses and we need a dump (log_error / panic ?) implementation for bolt_windows...

Reply:
True, added